enterprise/web.struts/src/org/netbeans/modules/web/struts/StrutsConfigUtilities.java

```diff
@@ -304,7 +304,7 @@ public static String[] getConfigFiles(FileObject dd){
             Servlet servlet = getActionServlet(dd);
             if (servlet!=null) {
                 InitParam[] params = servlet.getInitParam();
```

Comment:
How about rewriting it into

return Stream.of(servlet.getInitParam()).
   filter(p -> p.getParamName() != null).
   filter(p -> p.getParamName().startsWith(DEFAULT_MODULE_NAME)).
   flatMap(p -> Stream.of(p.getParamValue().split(",")).
   collect(toList())).
   toArray(new String[]{});